doap.rdf

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl"?>
+<rdf:RDF xml:lang="en"
+         xmlns="http://usefulinc.com/ns/doap#" 
+         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
+         xmlns:asfext="http://projects.apache.org/ns/asfext#"
+         xmlns:foaf="http://xmlns.com/foaf/0.1/">
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+   
+         https://www.apache.org/licenses/LICENSE-2.0
+   
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+  <Project rdf:about="https://iceberg.apache.org">
+    <created>2023-09-14</created>
+    <license rdf:resource="https://spdx.org/licenses/Apache-2.0" />
+    <name>Apache Iceberg</name>
+    <homepage rdf:resource="https://iceberg.apache.org" />
+    <asfext:pmc rdf:resource="https://iceberg.apache.org" />
+    <shortdesc>Iceberg is a high-performance format for huge analytic tables.</shortdesc>
+    <description>Iceberg brings the reliability and simplicity of SQL tables to big data, while making it possible for engines like Spark, Trino, Flink, Presto, Hive and Impala to safely work with the same tables, at the same time.</description>
+    <bug-database rdf:resource="https://github.com/apache/iceberg/issues" />
+    <mailing-list rdf:resource="https://iceberg.apache.org/community/" />
+    <download-page rdf:resource="https://iceberg.apache.org/releases/" />
+    <programming-language>Java</programming-language>
+    <programming-language>Python</programming-language>
```

Comment:
The Python code has been split out to: https://github.com/apache/iceberg-python

We also have the Go and Rust implementation:

- https://github.com/apache/iceberg-go
- https://github.com/apache/iceberg-rust

Reply:
Good point, let me update the `DOAP` accordingly.

Reply:
DOAP file only accept one Git repository, so I put `https://github.com/apache/iceberg`.

Reply:
I added all languages in the DOAP, and only the "main" repository.